vividus-plugin-web-app/src/main/java/org/vividus/selenium/WebDriverFactory.java

```diff
@@ -85,6 +86,12 @@ protected DesiredCapabilities updateDesiredCapabilities(DesiredCapabilities desi
                     {
                         WebDriverConfiguration configuration = getWebDriverConfiguration(type, false);
                         ChromeOptions chromeOptions = new ChromeOptions();
+                        Map<String, Object> prefs = new HashMap<>();
+                        Map<String, Object> profile = new HashMap<>();
+                        profile.put("cookie_controls_mode", 2);
+                        prefs.put("profile", profile);
+                        chromeOptions.setExperimentalOption("prefs", prefs);
```

Comment:
1) it should be configurable
2) we already have ability to set experimental options as per line 96:
```
web.driver.CHROME.experimental-options={...}
```

Reply:
thank you, done